info/v1/machine.go

```diff
@@ -228,6 +228,8 @@ type MachineInfo struct {
 
 	// ID of cloud instance (e.g. instance-1) given to it by the cloud provider.
 	InstanceID InstanceID `json:"instance_id"`
+
+	VMStat map[string]int `json:"vmstat_info"`
```

Comment:
As it is a map I suggests using plural form for the name of variable, e.g. `VMStats`.
I suggest also changing json tag to `vmstats` instead of `vmstat_info`.